packages/ag-charts-enterprise/src/features/context-menu/contextMenu.ts

```diff
@@ -148,8 +148,7 @@ export class ContextMenu extends _ModuleSupport.BaseModuleInstance implements _M
             id: 'download',
             label: 'Download',
             action: () => {
-                // TODO: chart name
-                this.scene.download('chart');
+                this.scene.download(ctx.chartService.title?.text ?? 'chart');
```

Comment:
Note: this is the key change. The rest of the PR is just refactoring `DataService` to a more general-purpose `ChartService` interface.